app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -40,11 +49,50 @@ public int getLevelsCount() {
         return levelsCount;
     }
 
-    public int getCurrentLevel() {
-        return currentLevel;
+    public int getCurrentLevelFromSharedPreferences() {
```

Comment:
getCurrentLevelFromSharedPreferences and getLastLevelFromSharedPreferences are only used inside this class, can be private.

Reply:
Makes sense. Fixed.